pybadges/__init__.py

```diff
@@ -181,11 +193,28 @@ def badge(
     if (left_link or right_link) and whole_link:
         raise ValueError(
             'whole_link may not bet set with left_link or right_link')
+
+    if show_trend and right_image:
+        raise ValueError('right-image and trend cannot be used together.')
+
+    if show_trend:
+        samples = show_trend if len(show_trend) <= 10 else show_trend[:10]
+        if len(samples) < 10:
+            samples = [0] * (10 - len(samples)) + samples
```

Comment:
Yep, that actually makes a lot of sense since we're anyways repeating same values. If we fall short (less than 10 numbers supplied), we can just repeat the same value more number of times.